### DISCUSSION TITLE
Colours

### DISCUSSION THREAD

**Discussion by StephenHillenbrand** [Victor-tafe/Tafe230231#5] [General]
Is there a specific colour being used for the background on all the Xaml pages?

while I'm thinking about it, what about the buttons are they different colour or standard?

**Comment by Victor-tafe** [comment]
I'm happy to keep the background colour as it is which is light blue. We can add a little colour to the buttons to make it pop.
________________________________
From: StephenHillenbrand ***@***.***>
Sent: Thursday, 21 September 2023 11:38 AM
To: Victor-tafe/Tafe230231 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Victor-tafe/Tafe230231] Colours (Discussion #5)


Is there a specific colour being used for the background on all the Xaml pages?

while I'm thinking about it, what about the buttons are they different colour or standard?
